WorkspaceServer.Tests/PackageRestoreContextTests.cs

```diff
@@ -12,13 +12,19 @@ namespace WorkspaceServer.Tests
     public class PackageRestoreContextTests
     {
         [Fact]
-        public async Task It_works()
+        public async Task Returns_new_references_if_they_are_added()
         {
-            using (var context = new PackageRestoreContext())
-            {
-                var refs = await context.AddPackage("FluentAssertions", "5.7.0");
-                refs.ToArray().Length.Should().Be(2);
-            }
+            var refs = await new PackageRestoreContext().AddPackage("FluentAssertions", "5.7.0");
+            refs.Should()
+                .Contain(r => r.Display.Contains("FluentAssertions.dll")
+                        || r.Display.Contains("System.Configuration.ConfigurationManager"));
```

Comment:
Why `||`? This smells of nondeterminism.

Reply:
FluentAssertions nuget package adds these 2 assemblies...

Reply:
The assertion will pass if only one is present though. Maybe `Should().BeEquivalentTo()`?

Reply:
The overloads there are awkward if you want to use a validation delegate so I've opted to call contains twice.